Src/FluentAssertions/Primitives/StringAssertions.cs

```diff
@@ -1114,6 +1114,86 @@ public AndConstraint<TAssertions> BeNullOrWhiteSpace(string because = "", params
             return new AndConstraint<TAssertions>((TAssertions)this);
         }
 
+        /// <summary>
+        /// Asserts that a string is uppercase.
```

Comment:
I think it would be better to phrase it like "Asserts that all characters in a string are in upper casing"